### ISSUE TITLE
loss = nan

### ISSUE THREAD

**Issue by zdyshine** [Sierkinhane/crnn_chinese_characters_rec#59]

**Comment by zdyshine** [comment]
您好，很高兴使用您的分享，但是训练到第轮的时候出现loss=nan的情况，请问您有什么建议吗？CTCLoss使用的是pytorch1.0中自带的损失函数。

**Comment by Sierkinhane** [comment]
调一下学习率，或者换优化器

**Comment by aabbcc1234mxd** [comment]
好像是bug，我遇到过，重新跑一下训练再试试

**Comment by zdyshine** [comment]
万分感谢您的回答。我把初始化学习率调整为之前的十分之一后可以训练了。但是在loss降到0.0几之后，loss又会等于nan了。就不太知道是什么情况了。您能在指点一下吗？




发自网易邮箱大师
在2019年3月16日 08:41，IdleCell.<notifications@github.com> 写道：

调一下学习率，或者换优化器

**Comment by LJXLJXLJX** [comment]
> 调一下学习率，或者换优化器

你好，请问为什么学习率会影响ctc loss呢，我用tensorflow版本也遇到了这个问题，学习率偏高的时候，ctc loss就一直是nan

**Comment by Sierkinhane** [comment]
explosion of gradients

**Comment by Sierkinhane** [comment]
#124
